matminer/featurizers/base.py

```diff
@@ -364,10 +364,35 @@ def featurize_many(self, entries, ignore_errors=False, return_errors=False,
                                            return_errors=return_errors,
                                            pbar=pbar)
             with Pool(self.n_jobs) as p:
-                func = partial(self.featurize_wrapper,
+                func = partial(self.featurize_imap_wrapper,
                                return_errors=return_errors,
                                ignore_errors=ignore_errors)
-                return p.map(func, entries, chunksize=self.chunksize)
+                entries = list(enumerate(entries))  # turns entries into [(0, entry_1), (1, entry_2), (2, entry_3), ... ]
```

Comment:
At this point entries is still wrapped by tqdm (see line 348). When you convert the tqdm function into a list it will produce a progress bar that goes from 0 to 100 % immediately. You then call tqdm again on line 379, thereby producing a secondary progress bar.

You need to rewrite the logic so that there is only one progress bar if p.imap_unordered is called. I.e. only call tqdm_func on entries in the `n_jobs == 1` or `sys.version_info[0] < 3` cases (provided `pbar == True`).